PagedRouteProviderInterface.php

```diff
@@ -1,4 +1,14 @@
 <?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
```

Comment:
The original had a duplicated header already, probably why we got an extra one here.